CMakeLists.txt

```diff
@@ -320,8 +320,10 @@ endif()
 if(QUIC_TLS STREQUAL "schannel")
     message(STATUS "Enabling Schannel configuration tests")
     list(APPEND QUIC_COMMON_DEFINES QUIC_TEST_SCHANNEL_FLAGS=1)
-    message(STATUS "Enabling UDP Send Queuing")
-    list(APPEND QUIC_COMMON_DEFINES CXPLAT_DATAPATH_QUEUE_SENDS=1)
+    if (NOT QUIC_SHARED_EC)
```

Comment:
There still might be reasons to batch call the sends over and over. Thoughts? Let's talk about this some. Ideally, I'd like to have this decision be data-driven.

Reply:
I can't really think of any reasons. Because we have a maximum batch size, the send loop is going to exit as soon as it calls into send. Then the next thing that will happen will be a IOCP wakeup just to process the send that was queued. The queueing was added to take advantage of the often not full datapath thread. But with shared ec, there is no datapath thread.